diff --git a/src/NzbDrone.Common/Instrumentation/NzbDroneLogger.cs b/src/NzbDrone.Common/Instrumentation/NzbDroneLogger.cs
index 436b65821..a1e4c1425 100644
--- a/src/NzbDrone.Common/Instrumentation/NzbDroneLogger.cs
+++ b/src/NzbDrone.Common/Instrumentation/NzbDroneLogger.cs
@@ -6,7 +6,6 @@
 using NLog.Targets;
 using NzbDrone.Common.EnvironmentInfo;
 using NzbDrone.Common.Extensions;
-using NzbDrone.Common.Instrumentation.Sentry;
 
 namespace NzbDrone.Common.Instrumentation
 {
@@ -41,8 +40,6 @@ public static void Register(IStartupContext startupContext, bool updateApp, bool
                 RegisterDebugger();
             }
 
-            RegisterSentry(updateApp);
-
             if (updateApp)
             {
                 RegisterUpdateFile(appFolderInfo);
@@ -62,36 +59,6 @@ public static void Register(IStartupContext startupContext, bool updateApp, bool
             LogManager.ReconfigExistingLoggers();
         }
 
-        private static void RegisterSentry(bool updateClient)
-        {
-            string dsn;
-
-            if (updateClient)
-            {
-                dsn = "https://f73eb750155b4636af91356e0ed756ea@sentry.servarr.com/20";
-            }
-            else
-            {
-                dsn = RuntimeInfo.IsProduction
-                    ? "https://52ab4ed8003a42f383b6d24d66daf317@sentry.servarr.com/15"
-                    : "https://0522924d625c497f86fc2a1b22aaf21d@sentry.servarr.com/16";
-            }
-
-            var target = new SentryTarget(dsn)
-            {
-                Name = "sentryTarget",
-                Layout = "${message}"
-            };
-
-            var loggingRule = new LoggingRule("*", updateClient ? LogLevel.Trace : LogLevel.Debug, target);
-            LogManager.Configuration.AddTarget("sentryTarget", target);
-            LogManager.Configuration.LoggingRules.Add(loggingRule);
-
-            // Events logged to Sentry go only to Sentry.
-            var loggingRuleSentry = new LoggingRule("Sentry", LogLevel.Debug, target) { Final = true };
-            LogManager.Configuration.LoggingRules.Insert(0, loggingRuleSentry);
-        }
-
         private static void RegisterDebugger()
         {
             DebuggerTarget target = new DebuggerTarget();
diff --git a/src/NzbDrone.Core/Instrumentation/ReconfigureLogging.cs b/src/NzbDrone.Core/Instrumentation/ReconfigureLogging.cs
index 38d4e627a..f8ccb8b5a 100644
--- a/src/NzbDrone.Core/Instrumentation/ReconfigureLogging.cs
+++ b/src/NzbDrone.Core/Instrumentation/ReconfigureLogging.cs
@@ -4,10 +4,8 @@
 using NLog.Config;
 using NLog.Targets.Syslog;
 using NLog.Targets.Syslog.Settings;
-using NzbDrone.Common.EnvironmentInfo;
 using NzbDrone.Common.Extensions;
 using NzbDrone.Common.Instrumentation;
-using NzbDrone.Common.Instrumentation.Sentry;
 using NzbDrone.Core.Configuration;
 using NzbDrone.Core.Configuration.Events;
 using NzbDrone.Core.Datastore;
@@ -62,9 +60,6 @@ public void Reconfigure()
             // Log Sql
             SqlBuilderExtensions.LogSql = _configFileProvider.LogSql;
 
-            // Sentry
-            ReconfigureSentry();
-
             LogManager.ReconfigExistingLoggers();
         }
 
@@ -99,16 +94,6 @@ private void SetLogRotation()
             }
         }
 
-        private void ReconfigureSentry()
-        {
-            var sentryTarget = LogManager.Configuration.AllTargets.OfType<SentryTarget>().FirstOrDefault();
-            if (sentryTarget != null)
-            {
-                sentryTarget.SentryEnabled = (RuntimeInfo.IsProduction && _configFileProvider.AnalyticsEnabled) || RuntimeInfo.IsDevelopment;
-                sentryTarget.FilterEvents = _configFileProvider.FilterSentryEvents;
-            }
-        }
-
         private void SetSyslogParameters(string syslogServer, int syslogPort, LogLevel minimumLogLevel)
         {
             var syslogTarget = new SyslogTarget();
diff --git a/src/NzbDrone.Host/Startup.cs b/src/NzbDrone.Host/Startup.cs
index ccd9dcbdb..eabe1c324 100644
--- a/src/NzbDrone.Host/Startup.cs
+++ b/src/NzbDrone.Host/Startup.cs
@@ -19,7 +19,6 @@
 using Microsoft.OpenApi.Models;
 using NLog.Extensions.Logging;
 using NzbDrone.Common.EnvironmentInfo;
-using NzbDrone.Common.Instrumentation;
 using NzbDrone.Common.Processes;
 using NzbDrone.Common.Serializer;
 using NzbDrone.Core.Configuration;
@@ -211,7 +210,6 @@ public void ConfigureServices(IServiceCollection services)
                               Lazy<ILogDatabase> logDatabaseFactory,
                               DatabaseTarget dbTarget,
                               ISingleInstancePolicy singleInstancePolicy,
-                              InitializeLogger initializeLogger,
                               ReconfigureLogging reconfigureLogging,
                               IAppFolderFactory appFolderFactory,
                               IProvidePidFile pidFileProvider,
@@ -221,7 +219,6 @@ public void ConfigureServices(IServiceCollection services)
                               LidarrErrorPipeline errorHandler,
                               IEventAggregator eventAggregator)
         {
-            initializeLogger.Initialize();
             appFolderFactory.Register();
             pidFileProvider.Write();
 
