diff --git a/frontend/src/Components/Page/Header/PageHeader.js b/frontend/src/Components/Page/Header/PageHeader.js
index 0f5d7129a8..e455d21190 100644
--- a/frontend/src/Components/Page/Header/PageHeader.js
+++ b/frontend/src/Components/Page/Header/PageHeader.js
@@ -4,7 +4,6 @@ import keyboardShortcuts, { shortcuts } from 'Components/keyboardShortcuts';
 import IconButton from 'Components/Link/IconButton';
 import Link from 'Components/Link/Link';
 import { icons } from 'Helpers/Props';
-import translate from 'Utilities/String/translate';
 import ArtistSearchInputConnector from './ArtistSearchInputConnector';
 import KeyboardShortcutsModal from './KeyboardShortcutsModal';
 import PageHeaderActionsMenu from './PageHeaderActionsMenu';
@@ -75,23 +74,6 @@ class PageHeader extends Component {
         <ArtistSearchInputConnector />
 
         <div className={styles.right}>
-          <IconButton
-            className={styles.donate}
-            name={icons.HEART}
-            aria-label="Donate"
-            to="https://lidarr.audio/donate"
-            size={14}
-            title={translate('Donate')}
-          />
-
-          <IconButton
-            className={styles.translation}
-            title={translate('SuggestTranslationChange')}
-            name={icons.TRANSLATE}
-            to="https://translate.servarr.com/projects/servarr/lidarr/"
-            size={24}
-          />
-
           <PageHeaderActionsMenu
             onKeyboardShortcutsPress={this.onOpenKeyboardShortcutsModal}
           />
diff --git a/frontend/src/System/Status/Status.js b/frontend/src/System/Status/Status.js
index c1f5dac3bd..88135d5508 100644
--- a/frontend/src/System/Status/Status.js
+++ b/frontend/src/System/Status/Status.js
@@ -4,9 +4,7 @@ import PageContentBody from 'Components/Page/PageContentBody';
 import translate from 'Utilities/String/translate';
 import AboutConnector from './About/AboutConnector';
 import DiskSpaceConnector from './DiskSpace/DiskSpaceConnector';
-import Donations from './Donations/Donations';
 import HealthConnector from './Health/HealthConnector';
-import MoreInfo from './MoreInfo/MoreInfo';
 
 class Status extends Component {
 
@@ -20,8 +18,6 @@ class Status extends Component {
           <HealthConnector />
           <DiskSpaceConnector />
           <AboutConnector />
-          <MoreInfo />
-          <Donations />
         </PageContentBody>
       </PageContent>
     );
